include/enmasse-kubernetes.jsonnet

```diff
@@ -10,7 +10,7 @@ local flavorConfig = import "flavor.jsonnet";
     "items": [ templateConfig.generate(use_sasl, with_kafka, false),
                addressController.deployment("enmasseproject/address-controller", "false", 0),
                restapiRoute.ingress(""),
-               addressController.service,
+               addressController.service("LoadBalancer"),
```

Comment:
Yes, I'm planning to do that also, as I think most people want to use Ingress in a production setup, but need to think a bit about if we should make a completely separate template, or make the ingress part an 'add-on' template.